File: apps/android/app/src/main/java/com/questionpro/model/MenuType.java
package com.questionpro.model;

public enum MenuType{

    LOGOUT("logout"),
    SHARE("share"),
    LINK("link"),
    HOME("home"),
    APP_URL("appUrl"),
    DYNAMIC("dynamic"),
    DYNAMIC_WITH_BADGE("dynamicWithBadge");

    private String menuType="";
    MenuType(String menuType){
        this.menuType = menuType;
    }

    public String getValue(){
        return menuType;
    }

    public static MenuType resolve(String type){
        for(MenuType menuType: MenuType.values()){
            if(menuType.getValue().equalsIgnoreCase(type)){
                return menuType;
            }
        }
        return MenuType.LINK;
    }
  }

File: apps/ios/ios/GeneralUtilityModule/Model/MenuType.swift
//
//  MenuType.swift
//  QuestionProCommunities
//
//  Created by Jignesh Raiyani on 8/26/16.
//  Copyright © 2016 QuestionPro Pvt Ltd. All rights reserved.
//

import UIKit



enum MenuTypes : String {
    
    case LOGOUT = "Logout"
    case SHARE = "Share"
    case LINK = "Link"
    case HOME = "Home"
    
    var description: String {
        return self.rawValue
    }
}